### ISSUE TITLE
Bugs

### ISSUE THREAD

**Issue by carly20** [mattabernethy/comp580-project#3]
I was never penalized for selecting an incorrect answer or letting the time run out.

**Comment by DawnDuskStar** [comment]
The game does not work with a screen reader, nor does the health change for the boss.

**Comment by DawnDuskStar** [comment]
There is no audio with the game. Also, here is a place for free sounds. 
http://soundbible.com/
